utils/src/main/kotlin/OkHttpClientHelper.kt

```diff
@@ -120,23 +120,23 @@ object OkHttpClientHelper {
      * [Result] wrapping an [IOException] (which might be a [HttpDownloadError]) on failure.
      */
     fun downloadText(url: String): Result<String> {
-        val request = Request.Builder().get().url(url).build()
-        val response = runCatching { execute(request) }.getOrElse { return Result.failure(it) }
+        val getRequest = Request.Builder().get().url(url).build()
+        val responseForGetRequest = runCatching { execute(getRequest) }.getOrElse { return Result.failure(it) }
 
-        return response.use {
+        return responseForGetRequest.use { response ->
```

Comment:
Is the reason for adding `get` into request / response variable the avoiding of a naming conflict when finding a better name for `it` below? 

If so (and as I like just `request` `response` a bit more, how about extracting `runCatching { execute(getRequest) }.getOrElse { return Result.failure(it) }` into a function instead, e.g. `fun request.execute(): Response`.

The you can just write: 

```
val request = Request.Builder().get().url(url).build()

return request.execute().use { response ->
...
```

Reply:
Reading up the documentation of  [1] it seems that the closing (Closable) interface only affects the response body.
So, using `use` is only necessary for the body. If you just modify the line 136 to `val text = response.use { it.body?.string().orEmpty()` }, then you can keep the shorter variables `request` and `response` for the rest of the function scope.


[1] https://square.github.io/okhttp/4.x/okhttp/okhttp3/-response/

Reply:
> Is the reason for adding `get` into request / response variable the avoiding of a naming conflict when finding a better name for `it` below?

Yes.

> Reading up the documentation of [1] it seems that the closing (Closable) interface only affects the response body.

That was a very good hint actually, thanks for that! This makes the code more nicer.